packages/inter-protocol/src/vpool-xyk-amm/multipoolMarketMaker.js

```diff
@@ -323,17 +323,17 @@ const start = async (zcf, privateArgs, baggage) => {
 
   // shared AMM state that will be lexically accessible to all pools
   /** @type {AmmState} */
-  const ammState = {
+  // @ts-expect-error type confusion due to harden()
```

Comment:
This is a legit type error. `params` is not a `TypedParamManager`; it's just the `.readonly()` returned by one.

So line 63 should be,
```
params: import('@agoric/governance/src/contractGovernance/typedParamManager').Getters<import('./params.js').AmmParams>
```

While there consider terms for line 57,
```
* zcf: ZCF<AMMTerms>
 ```

Reply:
done.  Thanks.